sync/games/__init__.py

```diff
@@ -0,0 +1,8 @@
+# dynamically include all python modules in this directory in the module
+import os
+__all__ = []
+for module in os.listdir(os.path.dirname(__file__)):
```

Comment:
there's an implementation of this in common.py as well...  trivial enough that i'm not bothered by it having a different copy here though